### PR TITLE
docs: add information about inviting present members

### LINKED ISSUE #7426: Help Wanted: Can't invite a guest user who's already a member of the workspace
# Context

The Mattermost Product Documentation doesn't provide guidance on how to successfully resolve the in-product invitation error: "This person is already a member of the workspace. Invite them as a member instead of a guest."

# Documentation Request

Document how to interpret and resolve the error, as described below.

If you try inviting someone who already has an account in your Mattermost instance, invite them to a particular channel instead.

# QA Validation Criteria

In a generated docs preview:
- Visiting the docs page directly returns a clarification note at the bottom of the page about attempting to invite members and guests who already have accounts on the server.

### PR DESCRIPTION
**Pull request by CBID2**
#### Summary
There was information missing on how to successfully resolve the in-product invitation error: "This person is already a member of the workspace" in the [Add people to your workspace tutorial](https://docs.mattermost.com/collaborate/invite-people.html.). 
> [!NOTE]
> This is my contribution for Hacktoberfest. 

#### Ticket Link
Fixes #7426

### CODE REVIEW

File: source/collaborate/invite-people.rst [hunk 1]
```diff
@@ -78,5 +78,7 @@ You can share a unique URL that takes users to a Mattermost account creation pag
 
 .. note::
   
-  Users on the same Mattermost server can view the username and full name of other users so they can communicate with them directly, via direct messages, regardless of team membership.
+  Users on the same Mattermost server can view the username and full name of other users so they can communicate with them directly, via direct messages, regardless of team membership. 
 
+.. note::
+If you send an invite link to a team member that is already on the team the server, you will receive the message "This user is already a team member". Click on **Done** and invite them to a different channel instead.
```
Comment (cwarnermm) [inline source/collaborate/invite-people.rst:84]:
```suggestion
  - You can't invite someone to a Mattermost team, as a **member**, if that user is already a member of that team. 
  - You can't invite someone to a Mattermost team, as a **guest**, if that user already has a Mattermost account. Invite them to a team as a **member** instead.
```

**Review by cwarnermm — APPROVED** [review]
Thanks, @CBID2!

**Comment by Rutam21** [comment]
@lindy65 Please assign this PR to me for QA Testing. Thanks.

**Comment by lindy65** [comment]
Assigned to you @Rutam21

**Comment by Rutam21** [comment]
### The PR changes are visible in the preview environment. However, I would like to point out a few things for you. @cwarnermm, please provide your valuable input on this.

- **The current changes add a new `Note` section with the required information. But there's already another `Note` section just above this. If they're to be placed like this, then they should be merged into one `Note` section with bullet points. Any thoughts on this?**

Screenshot 2024-10-17 023252 (1)

- **I was going through the discussion in issue #7426 and I came across this comment of yours(https://github.com/mattermost/docs/issues/7426#issuecomment-2414003719) with the suggestion to add this information to the `Tip` section on the page. By any chance, did you mean the changes need to be added here?**

Screenshot 2024-10-17 023420 (1)

cc: @lindy65 @cwarnermm

**Comment by cwarnermm** [comment]
@Rutam21 - Your attention to detail is excellent. As I reviewed this docs page and the incoming PR content changes, I realized that this page hasn't been through a technical review in quite some time. I've scheduled this page for a thorough review and update in a future release. For now, we'll accept the PR contribution as-is.

**Comment by CBID2** [comment]
@cwarnermm, can you put the hacktoberfest-accepted label on my PR? :) I want to ensure that it gets accepted for Hacktoberfest

**Comment by cwarnermm** [comment]
@CBID2 I don't believe the ``hacktoberfest-accepted`` label is needed in order for work to count towards the event. I'll confirm, though, and add where missing if needed.

**Pull request merged**